### PR TITLE
Add Move Analyzer to Install Sui

### PR DESCRIPTION
**Pull request by Clay-Mysten**
Add new Move Analyzer cargo install command directly to Install Sui

### CODE REVIEW

**Review by huitseeker — APPROVED** [review]
👍

File: doc/src/build/install.md [hunk 1]
```diff
@@ -29,14 +29,22 @@ $ echo $PATH
...
 
 ## IDE
-For Move development, we recommend the [Visual Studio Code (vscode)](https://code.visualstudio.com/) IDE with the [Move Analyzer](https://marketplace.visualstudio.com/items?itemName=move.move-analyzer) plugin. See more [IDE options](https://github.com/MystenLabs/awesome-move#ides) in the [Awesome Move](https://github.com/MystenLabs/awesome-move) docs.
+For Move development, we recommend the [Visual Studio Code (vscode)](https://code.visualstudio.com/) IDE with the Move Analyzer language server plugin installed:
+
+```shell
+$ cargo install --git https://github.com/diem/move move-analyzer
+```
+
+Then follow the Visual Studio Marketplace instructions to install the [Move Analyzer](https://marketplace.visualstudio.com/items?itemName=move.move-analyzer) extension.
```
Comment (awelc) [inline doc/src/build/install.md:38]: I would also mention that the instructions on installing move-analyzer that are available in Visual Studio Marketplace are outdated (and that's why one should follow instructions we provide here).

**Review by awelc — APPROVED** [review]
LGTM (with one comment)

File: doc/src/build/install.md (hunk 1 above)
Comment (Clay-Mysten) [reply doc/src/build/install.md]: > I would also mention that the instructions on installing move-analyzer that are available in Visual Studio Marketplace are outdated (and that's why one should follow instructions we provide here).

Done.

**Pull request merged**